src/nvim/hardcopy.c

```diff
@@ -358,8 +357,8 @@ static long_u prt_get_term_color(int colorindex)
 static void prt_get_attr(int hl_id, prt_text_attr_T *pattr, int modec)
 {
   int colorindex;
-  long_u fg_color;
-  long_u bg_color;
+  uint32_t fg_color;
+  uint32_t bg_color;
```

Comment:
Variable `bg_color` is not needed. We could directly set `pattr->bg_color = PRCOLOR_WHITE` at the end of the function.